lab-ron/entry.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
```

Comment:
Entry files are for backend code only. They should only have require statements. Everything else should be broken out into a separate, appropriately-named file and required in.